R/createReport.R

```diff
@@ -768,6 +769,50 @@ createReport = function(txt_folder = NULL, mztab_file = NULL, yaml_obj = list(),
   #save(file = rprt_fns$R_plots_file, list = "GPL")
   #cat(" done\n")
   
+  ##save required data of each metric in JSON-format
+  cat("Creating metric json file ...", '\n')
+  ##load the cv Term
+  cv <- read_cvTerm(txt_files$cv)
+  ##copy cv Term table
+  metric_info <- data.frame(matrix(unlist(cv), ncol = 4))
+  colnames(metric_info) <- c("cvRef", " accession", "name", "unit")
+  metric_info$value <- NA
+  metric_info$quality_type <- NA ##runQuality or setQuality
+  metric_info$input_file <- NA
+  ##Read the value stored in each class
+  metrics_file = rprt_fns$metrics_file
+  metric_info = readvalue(qcMetric = lst_qcMetrics, data = metric_info)
+  
+  inputfiles_info <- data.frame(location = c(txt_files$param, txt_files$summary, txt_files$groups, txt_files$evd,
```

Comment:
inputfiles should be the `.raw` or `.mzML` files (i.e. use the `raw.file` column from df_evidence or so to get them.

Reply:
That I misunderstood before, I always thought that inputFiles refers to those .txt files in the txt folder(like summary.txt...)